v2/datastream-to-spanner/terraform/samples/all-infrastructure/README.md

```diff
@@ -0,0 +1,554 @@
+## Sample Scenario: MySQL to Spanner using MySQL source configuration
+
+> **_SCENARIO:_** This Terraform example illustrates launching a live migration
```

Comment:
nit: please rename this as a mysql-production-migration. 

That is closer to what people would be looking for which this example aims to answer.

Reply:
Done!